src/v2/view-builder/views/ov/EnrollOVWithEmailChannelView.js

```diff
@@ -0,0 +1,23 @@
+import { loc } from 'okta';
```

Comment:
EnrollOVWithEmailChannelView.js , EnrollOVWithSmsChannelView.js both files have similar code, should we create some Factory class that can generate similar views but different settings

Reply:
These files are already using the baseView there is very little customization. I wil try to see if it makes to extract out when I add the footer links in https://oktainc.atlassian.net/browse/OKTA-320368

Reply:
Merged into one view :)